tests/integration/test_interaction.py

```diff
@@ -581,3 +581,138 @@ def test_keyboard_help(self):
         # Test mouse and keyboard interaction
         self.interact_with_keyboard_help(scroll_down=900)
         self.interact_with_keyboard_help(scroll_down=0, use_keyboard=True)
+
+
+class ZoneAlignInteractionTest(InteractionTestBase, BaseIntegrationTest):
+    """
+    Verifying Drag and Drop XBlock interactions using zone alignment.
+    """
+    PAGE_TITLE = 'Drag and Drop v2'
+    PAGE_ID = 'drag_and_drop_v2'
+
+    def setUp(self):
+        super(ZoneAlignInteractionTest, self).setUp()
+
+    def _get_scenario_xml(self):
+        return self._get_custom_scenario_xml("data/test_zone_align.json")
+
+    def _assert_zone_align_item(self, item_id, zone_id, align, action_key=None):
+        """
+        Test items placed in a zone with the given align setting.
+        Ensure that they are children of the zone.
+        """
+        # parent container has the expected alignment
+        zone_selector = "div[data-zone='%s'] .item-align" % zone_id
+        self.assertEquals(self._get_style(zone_selector, 'textAlign'), align)
+
+        # Items placed in zones with align setting are children of the zone
+        zone_item = '%s .option' % zone_selector
+        prev_placed_items = self._page.find_elements_by_css_selector(zone_item)
+
+        self.place_item(item_id, zone_id, action_key)
+        placed_items = self._page.find_elements_by_css_selector(zone_item)
+        self.assertEquals(len(placed_items), len(prev_placed_items) + 1)
+
+        # Not children of the target
+        target_item = '.target > .option'
+        self.assertEquals(len(self._page.find_elements_by_css_selector(target_item)), 0)
+
+        # Aligned items are relative positioned, with no transform or top/left
+        self.assertEquals(self._get_style(zone_item, 'position'), 'relative')
+        self.assertEquals(self._get_style(zone_item, 'transform'), 'none')
+        self.assertEquals(self._get_style(zone_item, 'left'), '0px')
+        self.assertEquals(self._get_style(zone_item, 'top'), '0px')
+
+        # Center-aligned items are display block
+        if align == 'center':
+            self.assertEquals(self._get_style(zone_item, 'display'), 'block');
+        # but other aligned items are just inline-block
+        else:
+            self.assertEquals(self._get_style(zone_item, 'display'), 'inline-block');
+
+    def test_no_zone_align(self):
+        """
+        Test items placed in a zone with no align setting.
+        Ensure that they are children of div.target, not the zone.
+        """
+        zone_id = "Zone No Align"
+        self.place_item(0, zone_id)
+        zone_item = "div[data-zone='%s'] .item-align .option" % zone_id
+        self.assertEquals(len(self._page.find_elements_by_css_selector(zone_item)), 0)
+
+        target_item = '.target > .option'
+        placed_item = self._page.find_elements_by_css_selector(target_item)
```

Comment:
@pomegranited Since you're using `find_elements_by_css_selector` here you could rename this to `placed_items`.